app/controllers/users/invitations_controller.rb

```diff
@@ -21,22 +21,27 @@ def create
       return
     end
 
-    invite_user(organisation)
+    add_user_to_organisation(organisation)
 
     redirect_to(after_path(organisation), notice: "#{invited_user.email} has been invited to join #{organisation.name}")
   end
 
 private
 
-  def invite_user(organisation)
+  def add_user_to_organisation(organisation)
     membership = invited_user.memberships.find_or_create_by(invited_by_id: current_user.id, organisation: organisation)
     membership.update_attributes(can_manage_team: params[:can_manage_team], can_manage_locations: params[:can_manage_locations])
-
-    send_invitation_email(membership)
+    send_membership_invite_to_confirmed_users_only(membership)
   end
 
-  def send_invitation_email(membership)
-    AuthenticationMailer.membership_instructions(invited_user, membership.invitation_token, organisation: current_organisation).deliver_now
+  def send_membership_invite_to_confirmed_users_only(membership)
```

Comment:
The plural 'users' sounds like it could send this out to more than one user.
What about 'notify_invited_to_team' ?

Just thinking out loud here, but this name applies only to existing users.
You won't be invited to a team if you created the account yourself.

Reply:
Yeah makes sense. But I want to have really well explained methods in this controller. Its already super confusing.